docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -269,6 +269,18 @@ passed to subsequent filters. Meaning, the following Lua code is invalid:
 *headers* is a table of key/value pairs to send. Note that the *:status* header
 must be set. *body* is a string and supplies the optional response body. May be nil.
 
+metadata()
+^^^^^^^^^^
+
+.. code-block:: lua
+
+  metadata = handle:metadata()
+
+Returns the current route entry metadata. Note that the metadata should be specified
+under the filter name.
```

Comment:
Can you further document how this is done, potentially with a small config snippet?